### PR TITLE
En 931 authoring users

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/api/roles.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ROLES } from 'test/mocks/roles';
+
+// eslint-disable-next-line
+export const getApiRoles = () => (
```
Comment (nicpuddu) [inline src/api/roles.js]: always specify which rule is being disabled

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/api/roles.js (hunk 1 above)
Comment (DMinnai) [reply src/api/roles.js]: OK fixed

**Comment by nicpuddu** [comment]
this will be merged while @DMinnai works on implementing `apiManager` on roles

**Comment by entando-service** [comment]
All is well

**Pull request merged**